### PR TITLE
chore/deploy: make interval property in PodMonitor configurable

### LINKED ISSUE #1638: Make `interval` property in PodMonitor configurable
**What would you like to be added**:

To [interval value in PodMonitor's `spec.podMetricsEndpoints`](https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/deployment/helm/node-feature-discovery/templates/prometheus.yaml#L15) configurable from `values.yaml`.

**Why is this needed**:

Being able to fine tune the interval. Current value might be too aggressive or not, depending on the cluster, but also specific needs for monitoring.

This should be a very simple feature to implement, with no impact in current installations.

**I'm open to contribute if required**.

### PR DESCRIPTION
**Pull request by TessaIO**
Resolves #1638

### CODE REVIEW

File: deployment/helm/node-feature-discovery/values.yaml [hunk 1]
```diff
@@ -537,4 +537,5 @@ tls:
 
 prometheus:
   enable: false
+  interval: 10s
```
Comment (marquiz) [inline deployment/helm/node-feature-discovery/values.yaml:540]: I'd suggest a small clarification on what interval are we configuring

```suggestion
  scrapeInterval: 10s
```

An alternative could be making deepening the hierarchy like `prometheus.podmonitor.interval` but I don't like that much, especially as we don't have other podmonitor settings

**Review by marquiz — COMMENTED** [review]
Thx @TessaIO. Small suggestion regarding naming.

We also need to document the new setting in `docs/deployment/helm.md`

/cc @cmontemuino

**Review by marquiz — APPROVED** [review]
Thanks @TessaIO, looks good to me, now

**Comment by TessaIO** [comment]
ping @ArangoGutierrez

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**